Mathlib/NumberTheory/PythagoreanTriples.lean

```diff
@@ -465,7 +457,6 @@ theorem isPrimitiveClassified_aux (hc : x.gcd y = 1) (hzpos : 0 < z) {m n : ℤ}
   norm_cast
 #align pythagorean_triple.is_primitive_classified_aux PythagoreanTriple.isPrimitiveClassified_aux
 
-/- ./././Mathport/Syntax/Translate/Tactic/Lean3.lean:132:4: warning: unsupported: rw with cfg: { occs := occurrences.pos[occurrences.pos] «expr[ ,]»([2, 3]) } -/
```

Comment:
This is telling you why the proof broke.

Reply:
The proof already diverges at `field_simp [hQ q]`, before the `rw` that the mathport comment refers. Checking in Lean 3, after `field_simp [hQ q]` the goal looks like
```lean
2 * ↑(q.denom) * ↑n * (1 + q ^ 2) = 2 * q * (↑(q.denom) ^ 2 + ↑n ^ 2)
```

But in the original Lean 4 file (before all our edits), after `field_simp [hQ q]` the goal looks like
```lean
2 * ↑(↑x / (↑z * (↑y / ↑z + 1))).den * ↑(↑x / (↑z * (↑y / ↑z + 1))).num =
  2 * ↑(↑x / (↑z * (↑y / ↑z + 1))).num * (↑(↑x / (↑z * (↑y / ↑z + 1))).den ^ 2 + ↑(↑x / (↑z * (↑y / ↑z + 1))).num ^ 2) /
    (↑(↑x / (↑z * (↑y / ↑z + 1))).den *
      (1 + ↑(↑x / (↑z * (↑y / ↑z + 1))).num ^ 2 / ↑(↑x / (↑z * (↑y / ↑z + 1))).den ^ 2))
```